storage/kubernetes/storage.go

```diff
@@ -40,6 +40,10 @@ const (
 	resourceDeviceToken     = "devicetokens"
 )
 
+const (
+	gcResultLimit = 10000
```

Comment:
Let's make it 500 as in Kubernetes client-go 
https://github.com/kubernetes/client-go/blob/8344b1c856d6484a78135387551c2c88a3f65a1e/tools/pager/pager.go#L31

Reply:
Sure.